test/regression/models/base/listeners_spec.js

```diff
@@ -23,7 +23,7 @@ describe('Models: listeners', function () {
 
     before(testUtils.teardownDb);
 
-    beforeEach(testUtils.setup('owner', 'settings'));
+    beforeEach(testUtils.setup('owner:pre', 'settings'));
```

Comment:
This was necessary because it uses `initOwnerUser()` which actually creates the roles compared to `insertOwnerUser()` which doesn't. Alternative would be to use `testUtils.set('roles', 'owner', 'settings').

The test started failing because we use `User.getOwnerUser()` in the `populateDefaults()` method and it relies on having the correct role record and relations set up.